### ISSUE TITLE
add info about default SQL to create fields and database related topics

### ISSUE THREAD

**Issue by DavidBruchmann** [TYPO3-Documentation/TYPO3CMS-Reference-TCA#1044]
### Current behavior

SQL syntax to create common fields is not mentioned at all, and neither some side aspects like collation of whole tables or single fields.

### Expected behavior/output

A basic chapter about default settings for fields, going into details about charset and collation, as well as special fields like timestamp in contrast to datetime, or json.  
Also some special scenarios should be explained with respect to the database, like languages or workspaces.

In the overview of single fields I'd wish that the default SQL for each field is listed (it might consider different sorts of databases like mySQL, msSQL, postgres, SqLite if required.

**Comment by DavidBruchmann** [comment]
special issues like changing select fields from int to string should be part of the general chapter too.

**Comment by sbuerk** [comment]
Will take care of this the next days.

**Comment by DavidBruchmann** [comment]
Yes, it's about `Default Schema generation for supported TCA types`.  
Great that you assigned yourself, curious what's coming!!!
